src/website/views/components/tabs/_examples.pug

```diff
@@ -58,23 +58,46 @@ h2 Examples
               .-text Tab 3 content
       .example-tabs.-pl--2
         ul.chi-tabs#example-base(role="tabpanel")
-          li.-disabled
+          li.-active
```

Comment:
You have added just the code snippets, should we change our current examples with the web components as well? @jllr